src/components/Table.tsx

```diff
@@ -82,12 +84,13 @@ export default function Table({
           type: 'inactive',
         });
       }
+      console.info(attendance);
       return (
         <div css={tableChicletsCss}>
-          {attendance.map(({ icon, text, type }, index) => (
+          {attendance.map(({ icon, text, type, noTranslate }, index) => (
             <span css={tableChicletCss(type)} key={index}>
               <Icon icon={icon} size={18} />
-              {text && <span>{text}</span>}
+              {text && <span className={(true === noTranslate) ? 'notranslate' : ''}>{text}</span>}
```

Comment:
minor nit: i might write this as `className={noTranslate ? 'notranslate' : undefined}` so react doesn't render the empty property

Reply:
Didn't realize undefined stripped the property, thanks - updated.